Moving to Spring Boot XML-less config / Code cleanup

srideepprasad opened the pull request:
Moved to spring boot + maven (since the BC folks use Maven)
Removed un-necessary "noise". Got rid off un-wanted framework dependencies (Sitemesh etc)
Deleted checked in JARs. 
Write some representative tests . Kept some missing (as before)

srideepprasad committed:
Moving codebase to spring boot + maven. Deleted un-necessary files

srideepprasad committed:
Moving to a full XML-less config

srideepprasad committed:
Changed Home URL from /home to /

srideepprasad committed:
Added Google Guava. Added some representative tests. Included a readme

srideepprasad committed:
Added readme

srideepprasad committed:
Minor refactoring - removed un-needed annotations. Tweaked resource mapping

srideepprasad committed:
Updating POM file

srideepprasad committed:
Deleting IDE related files

sunitparekh commented:
pull request merged and now you are collaborator for the repo, feel free to
update directly :-)

On Sat, Jan 24, 2015 at 9:30 AM, Srideep Prasad notifications@github.com
wrote:

> Moved to spring boot + maven (since the BC folks use Maven)
> Removed un-necessary "noise". Got rid off un-wanted framework dependencies
> (Sitemesh etc)
> Deleted checked in JARs.
> 
> ## Write some representative tests . Kept some missing (as before)
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/sunitparekh/videoworld/pull/1
> Commit Summary
> - Moving codebase to spring boot + maven. Deleted un-necessary files
> - Moving to a full XML-less config
> - Changed Home URL from /home to /
> - Added Google Guava. Added some representative tests. Included a
>   readme
> - Added readme
> - Minor refactoring - removed un-needed annotations. Tweaked resource
>   mapping
> - Updating POM file
> - Deleting IDE related files
> 
> File Changes
> - _D_ .classpath
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-0 (50)
> - _D_ .idea/.name
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-1 (1)
> - _D_ .idea/ant.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-2 (7)
> - _D_ .idea/compiler.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-3 (21)
> - _D_ .idea/copyright/profiles_settings.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-4 (5)
> - _D_ .idea/encodings.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-5 (5)
> - _D_ .idea/jsLibraryMappings.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-6 (3)
> - _D_ .idea/misc.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-7 (34)
> - _D_ .idea/modules.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-8 (9)
> - _D_ .idea/vcs.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-9 (7)
> - _D_ .project
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-10 (17)
> - _A_ README.md
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-11 (32)
> - _A_ Readme.MD
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-12 (0)
> - _D_ build.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-13 (209)
> - _D_ lib/build/servlet-api-2.5-20081211.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-14 (0)
> - _D_ lib/runtime/cglib-nodep-2.1_3.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-15 (0)
> - _D_ lib/runtime/commons-collections-3.2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-16 (0)
> - _D_ lib/runtime/commons-fileupload-1.2.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-17 (0)
> - _D_ lib/runtime/commons-lang-2.5.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-18 (0)
> - _D_ lib/runtime/freemarker-2.3.15.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-19 (0)
> - _D_ lib/runtime/joda-time-1.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-20 (0)
> - _D_ lib/runtime/ognl-2.7.3.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-21 (0)
> - _D_ lib/runtime/org.springframework.config.java-1.0.0.M4.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-22 (0)
> - _D_ lib/runtime/sitemesh-2.4.2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-23 (0)
> - _D_ lib/runtime/spring-beans-2.5.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-24 (0)
> - _D_ lib/runtime/spring-context-2.5.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-25 (0)
> - _D_ lib/runtime/spring-core-2.5.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-26 (0)
> - _D_ lib/runtime/spring-web-2.5.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-27 (0)
> - _D_ lib/runtime/struts2-core-2.1.8.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-28 (0)
> - _D_ lib/runtime/struts2-sitemesh-plugin-2.1.8.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-29 (0)
> - _D_ lib/runtime/struts2-spring-plugin-2.1.8.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-30 (0)
> - _D_ lib/runtime/xwork-core-2.1.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-31 (0)
> - _D_ lib/tools/ant-1.6.5.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-32 (0)
> - _D_ lib/tools/ant-contrib-1.0b3.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-33 (0)
> - _D_ lib/tools/commons-codec-1.4.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-34 (0)
> - _D_ lib/tools/commons-httpclient-3.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-35 (0)
> - _D_ lib/tools/commons-io-1.4.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-36 (0)
> - _D_ lib/tools/commons-logging-1.1.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-37 (0)
> - _D_ lib/tools/hamcrest-core-1.2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-38 (0)
> - _D_ lib/tools/hamcrest-library-1.2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-39 (0)
> - _D_ lib/tools/htmlunit-2.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-40 (0)
> - _D_ lib/tools/htmlunit-core-js-2.6.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-41 (0)
> - _D_ lib/tools/jetty-6.1.22.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-42 (0)
> - _D_ lib/tools/jetty-util-6.1.22.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-43 (0)
> - _D_ lib/tools/jsp-2.1-glassfish-9.1.1.B60.25.p2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-44 (0)
> - _D_ lib/tools/jsp-2.1-jetty-6.1.22.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-45 (0)
> - _D_ lib/tools/jsp-api-2.1-glassfish-9.1.1.B60.25.p2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-46 (0)
> - _D_ lib/tools/junit-dep-4.8.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-47 (0)
> - _D_ lib/tools/mockito-core-1.8.4.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-48 (0)
> - _D_ lib/tools/objenesis-1.2.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-49 (0)
> - _D_ lib/tools/serializer-2.7.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-50 (0)
> - _D_ lib/tools/xalan-2.7.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-51 (0)
> - _D_ lib/tools/xercesImpl-2.9.1.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-52 (0)
> - _D_ lib/tools/xml-apis-1.3.04.jar
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-53 (0)
> - _A_ pom.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-54 (91)
> - _A_ src/main/java/com/thoughtworks/videorental/Application.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-55 (18)
> - _A_ src/main/java/com/thoughtworks/videorental/Views.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-56 (19)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/LoginAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-57 (46)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/LogoutAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-58 (23)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/RentMoviesAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-59 (73)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/ViewAdminAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-60 (26)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/ViewCurrentRentalsAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-61 (38)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/ViewHistoryAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-62 (35)
> - _D_
>   src/main/java/com/thoughtworks/videorental/action/ViewHomeAction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-63 (25)
> - _A_
>   src/main/java/com/thoughtworks/videorental/config/VideoRentalConfiguration.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-64 (38)
> - _A_
>   src/main/java/com/thoughtworks/videorental/config/security/SecurityConfiguration.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-65 (50)
> - _A_
>   src/main/java/com/thoughtworks/videorental/config/security/UserAuthenticationProvider.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-66 (53)
> - _A_
>   src/main/java/com/thoughtworks/videorental/controllers/AdminController.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-67 (28)
> - _A_
>   src/main/java/com/thoughtworks/videorental/controllers/HomeController.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-68 (32)
> - _A_
>   src/main/java/com/thoughtworks/videorental/controllers/LoginController.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-69 (32)
> - _A_
>   src/main/java/com/thoughtworks/videorental/controllers/RentalController.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-70 (80)
> - _M_ src/main/java/com/thoughtworks/videorental/domain/Customer.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-71 (7)
> - _M_ src/main/java/com/thoughtworks/videorental/domain/Movie.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-72 (3)
> - _M_ src/main/java/com/thoughtworks/videorental/domain/Rental.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-73 (2)
> - _M_
>   src/main/java/com/thoughtworks/videorental/domain/Transaction.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-74 (2)
> - _D_
>   src/main/java/com/thoughtworks/videorental/interceptor/CustomerAware.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-75 (7)
> - _D_
>   src/main/java/com/thoughtworks/videorental/interceptor/CustomerLoginInterceptor.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-76 (34)
> - _M_
>   src/main/java/com/thoughtworks/videorental/repository/SetBasedRentalRepository.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-77 (2)
> - _M_
>   src/main/java/com/thoughtworks/videorental/repository/SetBasedTransactionRepository.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-78 (2)
> - _D_
>   src/main/java/com/thoughtworks/videorental/spring/VideoRentalConfiguration.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-79 (87)
> - _A_
>   src/main/java/com/thoughtworks/videorental/utils/DateTimeProvider.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-80 (16)
> - _A_ src/main/resources/application.properties
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-81 (2)
> - _D_ src/main/resources/struts.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-82 (53)
> - _D_ src/main/webapp/WEB-INF/applicationContext.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-83 (9)
> - _D_ src/main/webapp/WEB-INF/decorators.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-84 (31)
> - _D_ src/main/webapp/WEB-INF/sitemesh.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-85 (18)
> - _D_ src/main/webapp/WEB-INF/web.xml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-86 (58)
> - _M_ src/main/webapp/admin.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-87 (3)
> - _D_ src/main/webapp/decorators/login.dec
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-88 (21)
> - _D_ src/main/webapp/decorators/main.dec
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-89 (25)
> - _M_ src/main/webapp/history.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-90 (5)
> - _M_ src/main/webapp/home.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-91 (5)
> - _A_ src/main/webapp/includes/common.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-92 (5)
> - _A_ src/main/webapp/includes/footer.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-93 (2)
> - _A_ src/main/webapp/includes/header.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-94 (7)
> - _M_ src/main/webapp/login.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-95 (6)
> - _M_ src/main/webapp/rentals.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-96 (6)
> - _M_ src/main/webapp/statement.ftl
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-97 (3)
> - _A_
>   src/test/java/com/thoughtworks/videorental/controllers/LoginControllerTest.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-98 (52)
> - _A_
>   src/test/java/com/thoughtworks/videorental/controllers/RentalControllerTest.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-99 (137)
> - _R_
>   src/test/java/com/thoughtworks/videorental/domain/CustomerTest.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-100 (0)
> - _R_
>   src/test/java/com/thoughtworks/videorental/domain/TransactionTest.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-101 (0)
> - _D_
>   src/unit/java/com/thoughtworks/videorental/action/RentMoviesActionTest.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-102
>   (107)
> - _D_
>   src/unit/java/com/thoughtworks/videorental/action/ViewAdminActionTest.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-103 (27)
> - _D_ src/util/java/com/thoughtworks/jetty/JettyServer.java
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-104 (98)
> - _D_ videorental.iml
>   https://github.com/sunitparekh/videoworld/pull/1/files#diff-105
>   (465)
> 
> Patch Links:
> - https://github.com/sunitparekh/videoworld/pull/1.patch
> - https://github.com/sunitparekh/videoworld/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sunitparekh/videoworld/pull/1.

## 

thanks & regards,
Sunit
parekh.sunit@gmail.com

srideepprasad commented:
No issues.. I just remembered today morning and just sent out the request
On Jan 24, 2015 11:12 AM, "Sunit Parekh" notifications@github.com wrote:

> pull request merged and now you are collaborator for the repo, feel free
> to
> update directly :-)
> 
> On Sat, Jan 24, 2015 at 9:30 AM, Srideep Prasad notifications@github.com
> 
> wrote:
> 
> > Moved to spring boot + maven (since the BC folks use Maven)
> > Removed un-necessary "noise". Got rid off un-wanted framework
> > dependencies
> > (Sitemesh etc)
> > Deleted checked in JARs.
> > 
> > ## Write some representative tests . Kept some missing (as before)
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/sunitparekh/videoworld/pull/1
> > Commit Summary
> > - Moving codebase to spring boot + maven. Deleted un-necessary files
> > - Moving to a full XML-less config
> > - Changed Home URL from /home to /
> > - Added Google Guava. Added some representative tests. Included a
> >   readme
> > - Added readme
> > - Minor refactoring - removed un-needed annotations. Tweaked resource
> >   mapping
> > - Updating POM file
> > - Deleting IDE related files
> > 
> > File Changes
> > - _D_ .classpath
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-0 (50)
> > - _D_ .idea/.name
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-1 (1)
> > - _D_ .idea/ant.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-2 (7)
> > - _D_ .idea/compiler.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-3 (21)
> > - _D_ .idea/copyright/profiles_settings.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-4 (5)
> > - _D_ .idea/encodings.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-5 (5)
> > - _D_ .idea/jsLibraryMappings.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-6 (3)
> > - _D_ .idea/misc.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-7 (34)
> > - _D_ .idea/modules.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-8 (9)
> > - _D_ .idea/vcs.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-9 (7)
> > - _D_ .project
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-10 (17)
> > - _A_ README.md
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-11 (32)
> > - _A_ Readme.MD
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-12 (0)
> > - _D_ build.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-13 (209)
> > - _D_ lib/build/servlet-api-2.5-20081211.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-14 (0)
> > - _D_ lib/runtime/cglib-nodep-2.1_3.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-15 (0)
> > - _D_ lib/runtime/commons-collections-3.2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-16 (0)
> > - _D_ lib/runtime/commons-fileupload-1.2.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-17 (0)
> > - _D_ lib/runtime/commons-lang-2.5.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-18 (0)
> > - _D_ lib/runtime/freemarker-2.3.15.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-19 (0)
> > - _D_ lib/runtime/joda-time-1.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-20 (0)
> > - _D_ lib/runtime/ognl-2.7.3.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-21 (0)
> > - _D_ lib/runtime/org.springframework.config.java-1.0.0.M4.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-22 (0)
> > - _D_ lib/runtime/sitemesh-2.4.2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-23 (0)
> > - _D_ lib/runtime/spring-beans-2.5.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-24 (0)
> > - _D_ lib/runtime/spring-context-2.5.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-25 (0)
> > - _D_ lib/runtime/spring-core-2.5.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-26 (0)
> > - _D_ lib/runtime/spring-web-2.5.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-27 (0)
> > - _D_ lib/runtime/struts2-core-2.1.8.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-28 (0)
> > - _D_ lib/runtime/struts2-sitemesh-plugin-2.1.8.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-29 (0)
> > - _D_ lib/runtime/struts2-spring-plugin-2.1.8.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-30 (0)
> > - _D_ lib/runtime/xwork-core-2.1.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-31 (0)
> > - _D_ lib/tools/ant-1.6.5.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-32 (0)
> > - _D_ lib/tools/ant-contrib-1.0b3.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-33 (0)
> > - _D_ lib/tools/commons-codec-1.4.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-34 (0)
> > - _D_ lib/tools/commons-httpclient-3.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-35 (0)
> > - _D_ lib/tools/commons-io-1.4.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-36 (0)
> > - _D_ lib/tools/commons-logging-1.1.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-37 (0)
> > - _D_ lib/tools/hamcrest-core-1.2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-38 (0)
> > - _D_ lib/tools/hamcrest-library-1.2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-39 (0)
> > - _D_ lib/tools/htmlunit-2.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-40 (0)
> > - _D_ lib/tools/htmlunit-core-js-2.6.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-41 (0)
> > - _D_ lib/tools/jetty-6.1.22.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-42 (0)
> > - _D_ lib/tools/jetty-util-6.1.22.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-43 (0)
> > - _D_ lib/tools/jsp-2.1-glassfish-9.1.1.B60.25.p2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-44 (0)
> > - _D_ lib/tools/jsp-2.1-jetty-6.1.22.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-45 (0)
> > - _D_ lib/tools/jsp-api-2.1-glassfish-9.1.1.B60.25.p2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-46 (0)
> > - _D_ lib/tools/junit-dep-4.8.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-47 (0)
> > - _D_ lib/tools/mockito-core-1.8.4.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-48 (0)
> > - _D_ lib/tools/objenesis-1.2.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-49 (0)
> > - _D_ lib/tools/serializer-2.7.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-50 (0)
> > - _D_ lib/tools/xalan-2.7.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-51 (0)
> > - _D_ lib/tools/xercesImpl-2.9.1.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-52 (0)
> > - _D_ lib/tools/xml-apis-1.3.04.jar
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-53 (0)
> > - _A_ pom.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-54 (91)
> > - _A_ src/main/java/com/thoughtworks/videorental/Application.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-55 (18)
> > - _A_ src/main/java/com/thoughtworks/videorental/Views.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-56 (19)
> > - _D_
> >   src/main/java/com/thoughtworks/videorental/action/LoginAction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-57 (46)
> > - _D_
> >   src/main/java/com/thoughtworks/videorental/action/LogoutAction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-58 (23)
> > - _D_
> >   src/main/java/com/thoughtworks/videorental/action/RentMoviesAction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-59 (73)
> > - _D_
> >   src/main/java/com/thoughtworks/videorental/action/ViewAdminAction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-60 (26)
> > - _D_
> > 
> > src/main/java/com/thoughtworks/videorental/action/ViewCurrentRentalsAction.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-61 (38)
> > - _D_
> >   src/main/java/com/thoughtworks/videorental/action/ViewHistoryAction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-62 (35)
> > - _D_
> >   src/main/java/com/thoughtworks/videorental/action/ViewHomeAction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-63 (25)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/config/VideoRentalConfiguration.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-64 (38)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/config/security/SecurityConfiguration.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-65 (50)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/config/security/UserAuthenticationProvider.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-66 (53)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/controllers/AdminController.java
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-67 (28)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/controllers/HomeController.java
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-68 (32)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/controllers/LoginController.java
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-69 (32)
> > - _A_
> > 
> > src/main/java/com/thoughtworks/videorental/controllers/RentalController.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-70 (80)
> > - _M_ src/main/java/com/thoughtworks/videorental/domain/Customer.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-71 (7)
> > - _M_ src/main/java/com/thoughtworks/videorental/domain/Movie.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-72 (3)
> > - _M_ src/main/java/com/thoughtworks/videorental/domain/Rental.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-73 (2)
> > - _M_
> >   src/main/java/com/thoughtworks/videorental/domain/Transaction.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-74 (2)
> > - _D_
> > 
> > src/main/java/com/thoughtworks/videorental/interceptor/CustomerAware.java
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-75 (7)
> > - _D_
> > 
> > src/main/java/com/thoughtworks/videorental/interceptor/CustomerLoginInterceptor.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-76 (34)
> > - _M_
> > 
> > src/main/java/com/thoughtworks/videorental/repository/SetBasedRentalRepository.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-77 (2)
> > - _M_
> > 
> > src/main/java/com/thoughtworks/videorental/repository/SetBasedTransactionRepository.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-78 (2)
> > - _D_
> > 
> > src/main/java/com/thoughtworks/videorental/spring/VideoRentalConfiguration.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-79 (87)
> > - _A_
> >   src/main/java/com/thoughtworks/videorental/utils/DateTimeProvider.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-80 (16)
> > - _A_ src/main/resources/application.properties
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-81 (2)
> > - _D_ src/main/resources/struts.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-82 (53)
> > - _D_ src/main/webapp/WEB-INF/applicationContext.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-83 (9)
> > - _D_ src/main/webapp/WEB-INF/decorators.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-84 (31)
> > - _D_ src/main/webapp/WEB-INF/sitemesh.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-85 (18)
> > - _D_ src/main/webapp/WEB-INF/web.xml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-86 (58)
> > - _M_ src/main/webapp/admin.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-87 (3)
> > - _D_ src/main/webapp/decorators/login.dec
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-88 (21)
> > - _D_ src/main/webapp/decorators/main.dec
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-89 (25)
> > - _M_ src/main/webapp/history.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-90 (5)
> > - _M_ src/main/webapp/home.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-91 (5)
> > - _A_ src/main/webapp/includes/common.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-92 (5)
> > - _A_ src/main/webapp/includes/footer.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-93 (2)
> > - _A_ src/main/webapp/includes/header.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-94 (7)
> > - _M_ src/main/webapp/login.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-95 (6)
> > - _M_ src/main/webapp/rentals.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-96 (6)
> > - _M_ src/main/webapp/statement.ftl
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-97 (3)
> > - _A_
> > 
> > src/test/java/com/thoughtworks/videorental/controllers/LoginControllerTest.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-98 (52)
> > - _A_
> > 
> > src/test/java/com/thoughtworks/videorental/controllers/RentalControllerTest.java
> > 
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-99 (137)
> > - _R_
> >   src/test/java/com/thoughtworks/videorental/domain/CustomerTest.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-100 (0)
> > - _R_
> >   src/test/java/com/thoughtworks/videorental/domain/TransactionTest.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-101 (0)
> > - _D_
> > 
> > src/unit/java/com/thoughtworks/videorental/action/RentMoviesActionTest.java
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-102
> > (107)
> > - _D_
> > 
> > src/unit/java/com/thoughtworks/videorental/action/ViewAdminActionTest.java
> > https://github.com/sunitparekh/videoworld/pull/1/files#diff-103 (27)
> > - _D_ src/util/java/com/thoughtworks/jetty/JettyServer.java
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-104 (98)
> > - _D_ videorental.iml
> >   https://github.com/sunitparekh/videoworld/pull/1/files#diff-105
> >   (465)
> > 
> > Patch Links:
> > - https://github.com/sunitparekh/videoworld/pull/1.patch
> > - https://github.com/sunitparekh/videoworld/pull/1.diff
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/sunitparekh/videoworld/pull/1.
> 
> ## 
> 
> thanks & regards,
> Sunit
> parekh.sunit@gmail.com
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sunitparekh/videoworld/pull/1#issuecomment-71301588.
